PendingReleaseNotes

```diff
@@ -166,57 +166,59 @@
 >=17.2.1
 
 * The "BlueStore zero block detection" feature (first introduced to Quincy in
-https://github.com/ceph/ceph/pull/43337) has been turned off by default with a
-new global configuration called `bluestore_zero_block_detection`. This feature,
-intended for large-scale synthetic testing, does not interact well with some RBD
-and CephFS features. Any side effects experienced in previous Quincy versions
-would no longer occur, provided that the configuration remains set to false.
-Relevant tracker: https://tracker.ceph.com/issues/55521
+  https://github.com/ceph/ceph/pull/43337) has been turned off by default. This
+  change was implemented with the introduction of a new global configuration
+  called ``bluestore_zero_block_detection``. This feature, intended for
+  large-scale synthetic testing, does not interact well with certain RBD and
+  CephFS features. Any side effects experienced in previous Quincy versions
+  will no longer occur, provided that the configuration remains set to
+  ``false``. Relevant tracker: https://tracker.ceph.com/issues/55521
 
-* telemetry: Added new Rook metrics to the 'basic' channel to report Rook's
-  version, Kubernetes version, node metrics, etc.
-  See a sample report with `ceph telemetry preview`.
-  Opt-in with `ceph telemetry on`.
+* telemetry: Added new Rook metrics to the ``basic`` channel to report Rook's
+  version, the Kubernetes version, node metrics, and other information.  To see
```

Comment:
@zdover23 - A double space seems to have been introduced during the line wrap procedure.